plugins/in_kafka/in_kafka.c

```diff
@@ -180,8 +180,12 @@ static int in_kafka_collect(struct flb_input_instance *ins,
 
         rd_kafka_message_destroy(rkm);
 
-        /* TO-DO: commit the record based on `ret` */
-        rd_kafka_commit(ctx->kafka.rk, NULL, 0);
+        
+        if(!ctx->enable_auto_commit) {
```

Comment:
Add a space here would be good to align our coding style.

```suggestion
        if (!ctx->enable_auto_commit) {
```